src/Microsoft.DotNet.Build.Tasks/PackageFiles/Symbols.targets

```diff
@@ -451,9 +451,9 @@
     $(WindowsPdbConversionTargetPath)
       * Location to place converted PDBs when using PortablePdbToConvertGlob. Recursive directory
         is preserved, if one is present in the glob.
-    @(ConversionOptions) [optional]
-      * ItemSpec: An entry in the Microsoft.DiaSymReader.Tools.PdbConversionOptions flags enum to
-        use for all conversions performed.
+    @(SuppressSourceLinkConversion) [optional]
+      * ItemSpec: An entry in the Microsoft.DiaSymReader.Tools.PortablePdbConversionOptions to indicate 
+        whether to include source link data as is for all conversions performed.
```

Comment:
I'd suggest:

```xml
$(SuppressSourceLinkConversion) [optional]
  * Whether to include source link data as is for all conversions performed.
```

Later, it should be passed as `SuppressSourceLinkConversion="$(SuppressSourceLinkConversion)"`, since it's a `bool` now.